GrappleGirlGame/Character.lua

```diff
@@ -44,35 +56,35 @@ function Character:update(dt)
 
     if (self.canJump and love.keyboard.isDown("space")) then
         self.canJump = false
-        self.body:applyLinearImpulse(0, -50)
+        self.body:applyLinearImpulse(0, -400)
     end
+    self:shootRope()
 end
 
-lastMouseState = {
-    b1 = false,
-    b2 = false,
-    b3 = false
-}
```

Comment:
There is an event for when the mouse is clicked or released. Use that instead of this. Additionally, if you're going to do it this way, it would be preferable to use it like an array: lastMouseState = {false, false, false}, and then you could index it with [1], [2], and [3].

Reply:
this is another old piece of code

Reply:
Still do use events instead of this system